### PR TITLE
Converted Spectre to Sass (SCSS).

### PR DESCRIPTION
**Pull request by samrawks**
Hello,

I've made a Sass version of spectre.

I've adjusted the gulpfile to better suit my own workflow. In Terminal on OS X you'd need to run _sudo npm install_ and then _sudo npm install gulp-sass_ inside the project's directory. From there just run the _gulp_ command and it'll watch for changes to SCSS files and update the CSS file automatically. Feel free to change the gulpfile back to the original if you prefer.

I wasn't sure if creating a pull request to the master was the best option but I couldn't see a sass branch available.

All the best,

### CODE REVIEW

**Comment by dak0rn** [comment]
Why do you need `sudo` for `npm`?

**Comment by samrawks** [comment]
@dak0rn you may or may not need to depending on your permissions. I just used the command given in the top answer [here](http://stackoverflow.com/questions/16151018/npm-throws-error-without-sudo) and fixed it for me. Thanks!

**Comment by dak0rn** [comment]
@samrawks Just asking because I use npm every day and it sounds like a broken install on your side.

**Comment by chrisssycollins** [comment]
I will totally contribute to the sass project!

File: src/_empty.scss [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
   .empty-title {
     font-size: 1.8rem;
     margin: 1.5rem 0 .5rem 0;
   }
   .empty-meta {
-    color: @core-meta-color;
+    color: $core-meta-color;
   }
-  .empty-action {
+  .btn {
```
Comment (chrisssycollins) [inline src/_empty.scss]: This should be left as .empty-action as it breaks the inline form with a button. 

screen shot 2016-06-04 at 17 02 30

File: package.json [hunk 2]
```diff
@@ -17,18 +17,14 @@
...
     "url": "https://github.com/picturepan2/spectre/issues"
   },
   "devDependencies": {
     "gulp": "^3.9.1",
-    "gulp-clean-css": "^2.0.6",
-    "gulp-csscomb": "^3.0.7",
-    "gulp-less": "^3.0.5",
-    "gulp-rename": "^1.2.2",
-    "less-plugin-autoprefix": "^1.5.1"
+    "gulp-sass": "^3.0.5"
```
Comment (DJTB) [inline package.json]: The latest gulp-sass is 2.3.1
This fails npm install.

Recommend `"gulp-sass": '^2'` instead.

File: src/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+$body-bg: #fff;
+$body-font-color: #333;
+$body-font-size: 1.4rem;
+$body-font-family: "Helvetica Neue", "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Hiragino Kaku Gothic Pro", Meiryo, "Malgun Gothic", sans-serif;
+
+// Sizes
+$layout-gutter: 1rem;
+
+// Colors
+$core-color: #5764c6;
```
Comment (Swizz) [inline src/_variables.scss]: Add the `!default` keyword at the end of variables to allow us to override them.

```sass
$core-color: red;
@import "spectre.css/src/spectre.scss";
```

[!default keyword on SASS doc](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variable_defaults_)

**Comment by eddiewang** [comment]
i'll help with this as well! what do we still need?

File: src/_empty.scss (hunk 1 above)
Comment (samrawks) [inline src/_empty.scss]: @chrisssycollins committed this back in. Thanks, good spot!

File: package.json (hunk 2 above)
Comment (samrawks) [inline package.json]: @DJTB switched this out to gulp-sass 2.3.2 and it working again for me. Let me know if it's still an issue.

File: src/_variables.scss (hunk 3 above)
Comment (samrawks) [inline src/_variables.scss]: @Swizz excellent point and added in. Thanks!

**Comment by nblackburn** [comment]
I have no issue with a SASS version but replacing the LESS version is just inconsiderate.

**Comment by samrawks** [comment]
@nblackburn no disrespect towards @picturepan2 or Spectre was intended here. Spectre is a great project. I simply prefer Sass over Less and wanted a version written to use the former. What I should have done though is create this in a separate branch rather than creating a pull request back into master.

**Comment by nblackburn** [comment]
@samrawks I know you mean the best, we just need to do what works best for everyone that way nobody has to lose out.

**Comment by samrawks** [comment]
@nblackburn you're totally right, it would be great if there were the option to choose to use whichever preprocessor you prefer.

**Comment by TigersWay** [comment]
+1 to keep a Less version. I have to totally agree with @nblackburn there is plenty of very good reasons not to destroy a nice work.
Once again Less and Sass are just 2 sides of the same coin. Don't let marketing wins over everything :-)

**Comment by luiguild** [comment]
Keep a LESS version, please.

**Pull request closed without merging**